source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -64,12 +70,15 @@ def make_params(self, request, instance):
 
                 # the geocodejson type for an "administrative_region" is "city"
                 if type == 'administrative_region':
-                    type = 'city'
-
-                types.append(type)
+                    types.append(self.TYPE_CITY)
+                elif type == 'address':
+                    types.extend([self.TYPE_STREET, self.TYPE_HOUSE])
+                elif type == 'stop_area':
+                    types.append(self.TYPE_STOP_AREA)
+                else:
+                    types.append(type)
```

Comment:
seems not working 
It concatenates each letter
`[u'p', u'u', u'b', u'l', u'i', u'c', u'_', u't', u'r', u'a', u'n', u's', ....`